script.js

```diff
@@ -37,3 +38,183 @@ open.addEventListener('click', () => {
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: './images/Snapshoot-Portfolio-2.png',
+    liveButton: 'See Live',
+    sourceButton: 'See Source',
+    live: 'images/live-version2.svg',
+    source: 'images/live-version.svg',
+  },
+  {
+    title: 'Facebook 360',
+    devs: ['FACEBOOK', '•', 'Full Stack Dev', '•', '2015'],
+    description: ['Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent, Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.', 'A daily selection of privately personalized reads; no accounts or sign-ups required.'],
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: './images/Snapshoot-Portfolio-3.png',
+    liveButton: 'See Live',
+    sourceButton: 'See Source',
+    live: 'images/live-version2.svg',
+    source: 'images/live-version.svg',
+  },
+  {
+    title: 'Uber Navigation',
+    devs: ['Uber', '•', 'Lead Developer', '•', '2018'],
+    description: ['Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent, Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.', 'A daily selection of privately personalized reads; no accounts or sign-ups required.'],
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: './images/Snapshoot-Portfolio4.png',
+    liveButton: 'See Live',
+    sourceButton: 'See Source',
+    live: 'images/live-version2.svg',
+    source: 'images/live-version.svg',
+  },
+];
+
+// Add project cards
+function show(index) {
+  const {
+    title,
+    devs,
+    description,
+    tags,
+    image,
+  } = projects[index];
+  const dev1 = devs[0];
+  const dev2 = devs[1];
+  const dev3 = devs[2];
+  const dev4 = devs[3];
+  const dev5 = devs[4];
+  const tag1 = tags[0];
+  const tag2 = tags[1];
+  const tag3 = tags[2];
+  const tag4 = tags[3];
```

Comment:
- [ ] [Optional] For best practices please don't repeat the code, in this case, you could use a forEach loop to create the variables, this will make your code clean